tools/roxvet/analyzers/validateimports/analyzer.go

```diff
@@ -329,7 +329,10 @@ func verifyImportsFromAllowedPackagesOnly(pass *analysis.Pass, imports []*ast.Im
 	}
 
 	if validImportRoot == "sensor/common" {
-		allowedPackages = appendPackageWithChildren(allowedPackages, "sensor/debugger") // Need this for unit tests
+		// Need this for unit tests.
+		allowedPackages = appendPackageWithChildren(allowedPackages, "sensor/debugger")
+		// Need this to connect to Scanner V4.
+		allowedPackages = appendPackageWithChildren(allowedPackages, "scanner/pkg/client")
```

Comment:
I think anything under `scanner/pkg` is fair game to use

Reply:
I thought pin-pointing to `scanner/pkg/client` could help block unnecessary dep on `scanner/` -- mainly because I don't see any other reason why the Sensor would depend on Scanner other than client-side code. If we want to create incompatible client packages in the future, we can version them, e.g., `scanner/pkg/client/v4` etc.

Reply:
@ross Is this resolved?